drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -294,6 +335,51 @@ local function temp_attr_handler(driver, device, ib, response)
   end
 end
 
+local mired_bounds_handler_factory = function(minOrMax)
+  return function(driver, device, ib, response)
+    if ib.data.value ~= nil then
+      if (ib.data.value < COLOR_TEMPERATURE_MIRED_MIN or ib.data.value > COLOR_TEMPERATURE_MIRED_MAX) then
+        device.log.warn_with({hub_logs = true}, string.format("Device reported a min or max color temperature %d mired outside of supported capability range", ib.data.value))
+        return
```

Comment:
Are we sure we want to ignore values outside of the range rather than clamp them? I feel like clamping might be better UX.

Reply:
These min and max values are defined based on the [colorTemperature's default range](https://github.ecodesamsung.com/iot/SmartThingsCapabilities/blob/8f6538bf03eacc140c38954299184bde9599886c/capabilities/color-temperature.src/color-temperature.yml#L11) of `1-30,000`. This is not a range that is used in practice, as most bulbs fall in the 2000K-8000K range. So, a value outside of this range is almost surely incorrect, and I think ignoring it is probably best for that reason. Additionally, a value of 0 mireds is impossible because it would cause a divide by zero error when converting to kelvin.

Reply:
Does the UI use the default range if this value gets dropped anyway? If so, then my point was moot anyway.

Reply:
Correct! The UI will use the default range of 1-30000 if we do not provide a value. At the moment, we do not set a value unless we have received a valid min and max value. I suppose we could set just one or the other if we do not receive a valid value for both by clamping the values as you mentioned above. That seems appropriate to me - it seems better to provide what we can even if we don't receive valid value for both min and max - would you agree?

Reply:
Yeah, I agree that seems like the best course of action.

Reply:
I updated both new handler functions to clamp the values as suggested. See 775564f06809491f7f3c5359f887bf7864edad48.